.github/workflows/ci.yml

```
@@ -41,6 +41,16 @@ jobs:
           sudo tlmgr install standalone preview doublestroke relsize fundus-calligra wasysym physics dvisvgm.x86_64-darwin dvisvgm rsfs wasy cm-super
           echo "::set-env name=PATH::$PATH"
 
+      - name: Install system dependencies (Windows)
+        if: runner.os == 'Windows'
+        run: |
+          Invoke-WebRequest https://yihui.org/gh/tinytex/tools/install-windows.bat -O install_tinytex.bat
```

```suggestion
          Invoke-WebRequest https://raw.githubusercontent.com/yihui/tinytex/master/tools/install-windows.bat -O install_tinytex.bat
```
Use GitHub links instead of other website links as it could be down.

---

According to [this](https://hostadvice.com/tools/whois/#yihui.org) analysis, TinyTeX's developer's site uses DigitalOcean to host, and CloudFlare for their CDN.

I really don't see an advantage to using raw GitHub links, since GitHub isn't a CDN anyway, and CloudFlare are probably much better at the job ;P
I'm open to changing it if necessary, but I don't see the need for it _as of now_.

---

AFAIK, what if the owner of that website either fails to pay for it? It is always better to use a more reliable CDN for it. The one we can always believe is GitHub. But we can also use other cdn which we know I will link below.
https://cdn.statically.io/gh/yihui/tinytex/7d5fa261/tools/install-windows.bat
https://ghcdn.rawgit.org/yihui/tinytex/master/tools/install-windows.bat
I tried jsdelivr but that doesn't accept batch files. I suggest this should be changed.

---

You've made your point. I find it highly unlikely that TinyTeX's dev would suddenly stop paying for the site they direct everyone who wants to learn about TinyTeX to, but hey, there _is_ a minuscule chance of that happening.

I'll see what the others have to say about this, just to get a second opinion, if that's fine by you.

---

Either way is fine by me.

---

@kilacoda suggested using try-except statements to make the github raw link a backup to yihui.org. This has been implemented in the latest commit.